Dispatchfile

```diff
@@ -0,0 +1,53 @@
+#!mesosphere/dispatch-starlark:v0.6
+# vi:syntax=python
+
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.4", "gitResource", "pullRequest", "push", "volume", "resourceVar", "secretVar")
+load("github.com/mesosphere/cluster-claim-controller/starlark/claim@master", "make_kind_cluster", "fetch_kubeconfig", "cleanup")
+
+
+git = "src-git"
+gitResource(git, url="$(context.git.url)", revision="$(context.git.commit)")
+
+cluster_name = "kubeaddons"
+kind_cluster=make_kind_cluster(cluster_name, git)
+
+task("integration-test", inputs=[git],
+     deps=[kind_cluster],
+     steps=[
+       fetch_kubeconfig(cluster_name, git),
+       k8s.corev1.Container(
+       name = "integration-tester",
+              workingDir = "/workspace/{}/".format(git),
+              image = "mesosphere/kubeaddons-ci:latest",
+              command = ["/bin/bash", "-c"],
+              args = [
+                  """
+                  set -euo pipefail
+                  export GIT_TERMINAL_PROMPT=1
+                  export GOPRIVATE=github.com/mesosphere/kubeaddons
+                  ssh-keyscan -t rsa github.com > /etc/ssh/known_hosts
+                  git config --global url."https://$GITHUB_TOKEN:@github.com/".insteadOf "https://github.com/"
+                  make kind-test
```

Comment:
do we want to spin up a kind cluster?
does the underlying node support the creation of kind?

```
 Cannot connect to the Docker daemon at unix:///var/run/docker.sock. 
```

Reply:
More recent kind should be playing nicely with podman too, if docker ends up being an issue for some reason? we might have luck there

Reply:
It's supposed to be using cluster claim controller to get kubeconfig of a kind cluster that is already running somewhere (from what I understand about cluster claim controller) so I'm not entirely sure why the container needs to be privileged.